### ISSUE TITLE
Create Dreams and Desires Greiflogger crash

### ISSUE THREAD

**Issue by Lunaticol** [LopyLuna/Create-Flavored#103]
Seems sometimes when chopping a tree down the server will crash
[crash-2024-09-14_22.00.38-server.txt](https://github.com/user-attachments/files/17003924/crash-2024-09-14_22.00.38-server.txt)

**Comment by LopyLuna** [comment]
try it with the mechanical saw

**Comment by Lunaticol** [comment]
alright

**Comment by LopyLuna** [comment]
any updates?

**Comment by Lunaticol** [comment]
yeah i shoved the item into item obliterator and there hasnt been anymore issues, I assume theres a problem when the item cuts down a tree and grieflogger trys to log it.

**Comment by LopyLuna** [comment]
welp that doesnt help to fix the issue for anyone else

**Comment by Lunaticol** [comment]
Well the Deforester Saw is whats causing the crashing.

**Comment by LopyLuna** [comment]
is it also reported to greiflogger?

**Comment by Lunaticol** [comment]
I havent opened an issue in grieflogger yet (I forgot)

**Comment by intervinn** [comment]
same here on forge 1.20.1
it crashes when using deforester saw, but doesn't when using mechanical saw (specifically for tall trees)
